tests/filecheck/lit.cfg

```diff
@@ -1,21 +1,14 @@
 import lit.formats
 import os
 
-config.name = "xDSL"
-config.test_format = lit.formats.ShTest()
-config.suffixes = ['.test', '.xdsl', '.mlir', '.py']
-
 config.test_source_root = os.path.dirname(__file__)
+xdsl_src = os.path.dirname(os.path.dirname(config.test_source_root))
 
-if "PYTHONPATH" in os.environ.keys():
-    config.environment["PYTHONPATH"] = config.test_source_root + "/../../xdsl:" + os.environ["PYTHONPATH"]
-else:
-    config.environment["PYTHONPATH"] = config.test_source_root + "/../../xdsl"
+config.name = "xDSL"
+config.test_format = lit.formats.ShTest(preamble_commands=[f"cd {xdsl_src}"])
```

Comment:
OMG, THIS IS IT!

I was looking for this thing for ages when configuring this the first time. That's why the design was so weird and kinda messed up. Nice, thanks a LOT for finding that.

Reply:
Thanks for appreciating it :pray: It is super confusing indeed!